docs/docs/guides/web3_plugin_guide/plugin_authors.md

```diff
@@ -13,14 +13,18 @@ To provide type safety and IntelliSense for your plugin users, please refer to t
 
 ## Plugin Dependencies
 
-At the minimum, your plugin should depend on the `4.x` version of `web3`. This will allow your plugin class to extend the provided `Web3PluginBase` abstract class. However, `web3` shouldn't be listed as a regular dependency, instead it should be listed in your plugin's `package.json` as a [peer dependency](https://nodejs.org/en/blog/npm/peer-dependencies/):
+At the minimum, your plugin should depend `web3` package version `4.0.2`. This will allow your plugin class to extend the provided `Web3PluginBase` abstract class. However, `web3` shouldn't be listed as a regular dependency, instead it should be listed in your plugin's `package.json` as a [peer dependency](https://nodejs.org/en/blog/npm/peer-dependencies/).
+
+:::important
+If the version `web3@4.0.2`, was not available yet. You can use the version `web3@4.0.2-dev.af57eae.0`.
+:::
```

Comment:
```suggestion
```

When this doc is officially release, version `4.0.2` (or maybe even `4.1.0`) will be the latest version available

Reply:
Yes, but after being merged with 4.x branch the documentation will be updated and people will read it before the release of 4.0.2 or 4.1.0, right?

Reply:
https://github.com/web3/web3.js/issues/6191 
I think lets not refer specific dev versions where possible in future, as these are unstable and only for dev testing.